build/manifest.ps1

```diff
@@ -35,11 +35,13 @@ $artifacts = @{
         "Microsoft.Quantum.QirGeneration",
         "Microsoft.Quantum.CSharpGeneration",
         "Microsoft.Quantum.DocumentationGenerator",
+        "Microsoft.Quantum.Telemetry",
         "Microsoft.Quantum.ProjectTemplates",
         "Microsoft.Quantum.Sdk"
     ) | ForEach-Object { Join-Path $Env:NUGET_OUTDIR "$_.$Env:NUGET_VERSION.nupkg" };
 
     Assemblies = $VsixAssemblies + @(
+        ".\src\Telemetry\Library\bin\$Env:BUILD_CONFIGURATION\netstandard2.1\Microsoft.Quantum.Telemetry.dll",
```

Comment:
This can probably fix the build.
```suggestion
        ".\src\Telemetry\Library\bin\$Env:BUILD_CONFIGURATION\netcoreapp3.1\Microsoft.Quantum.Telemetry.dll",
```
See `src/Telemetry/Library/Telemetry.csproj`: 
> \<TargetFramework\>**netcoreapp3.1**\</TargetFramework\>